app/models/article.rb

```diff
@@ -22,6 +22,7 @@ class Article < ApplicationRecord
   counter_culture :organization
 
   has_many :comments, as: :commentable, inverse_of: :commentable
+  has_many :profile_pins, as: :pinnable, inverse_of: :pinnable
```

Comment:
Why does an article has many profile_pins? I don't understand. Shouldn't an article have at most one pin?

Reply:
I didn't build in the full functionality yet, but created the structure so that one article could be simultaneously pinned to a user, an organization and any of its tags. Each acts as a "profile" in this relationship.